core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoInvalidCharactersNotice.java

```diff
@@ -0,0 +1,54 @@
+package org.mobilitydata.gtfsvalidator.notice;
+
+import static org.mobilitydata.gtfsvalidator.notice.SeverityLevel.WARNING;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidationNotice.UrlRef;
+
+/**
+ * This field contains invalid characters such as the replacement character (\uFFFD).
+ *
+ * <p>Fields with customer-facing text should not contain invalid characters to ensure good
+ * readability and accessibility.
+ *
+ * <table style="table-layout:auto; width:auto;">
+ *   <caption>Examples:</caption>
+ *   <tr>
+ *     <th><code>Field Text</code></th>
+ *     <th><code>Dataset</code></th>
+ *   </tr>
+ *   <tr>
+ *     <td>"Invalid�Character"</td>
+ *     <td><a href="http://example.com">Example Dataset</a></td>
+ *   </tr>
+ * </table>
+ */
+@GtfsValidationNotice(
+    severity = WARNING,
+    urls = {
+      @UrlRef(
+          label = "Best Practices for All Files",
+          url = "https://gtfs.org/schedule/reference/#file-requirements")
+    })
+public class NoInvalidCharactersNotice extends ValidationNotice {
```

Comment:
Rename the notice to:
```suggestion
public class InvalidCharactersNotice extends ValidationNotice {
```

Reply:
Good catch!